README.md

```diff
@@ -786,31 +788,69 @@ player.getCurrentChapter().then(function(chapter) {
 
 ### getColor(): Promise&lt;string, Error&gt;
 
-Get the color for this player.
-
+Get the accent color for this player. Note that this is deprecated in place of `getColors`.
 ```js
 player.getColor().then(function(color) {
     // color = the hex color of the player
 }).catch(function(error) {
     // an error occurred
 });
+
+```
+
+### getColors(): Promise&lt;string[], Error&gt;
+
+Get all colors used for this player.
+The return value is an array of colorOne, colorTwo, colorThree, and colorFour.
```

Comment:
`colorOne`, `colorTwo` etc. might not mean much to users - we should using consider product names.

Reply:
Good idea. Updated.